api/generator/generator.go

```diff
@@ -150,6 +151,7 @@ func (g *Handler) Handle(w http.ResponseWriter, r *http.Request) error {
 // sending the CSR to the server.
 type CertGeneratorHandler struct {
 	generator *csr.Generator
+	bundler   *bundler.Bundler
```

Comment:
Would it be easier to just use the API client to bundle the cert instead of adding a lot more parameters to the `NewCertGeneratorHandler`?